package.json

```diff
@@ -141,12 +141,15 @@
       "lcov"
     ],
     "collectCoverageFrom": [
-      "src/js/components/**/*.{js}",
-      "src/js/contexts/**/*.{js}",
-      "src/js/themes/**/*.{js}",
-      "src/js/utils/**/*.{js}",
-      "!src/js/components/**/*.{stories.js}",
-      "!src/js/components/**/stories/*.js"
+      "src/js/components/**/*.js",
+      "src/js/contexts/**/*.js",
+      "src/js/themes/**/*.js",
+      "src/js/utils/**/*.js",
+      "!src/js/**/**/*.stories.js",
+      "!src/js/**/**/stories/*.js",
+      "!src/js/**/**/stories/**/**/*.js",
```

Comment:
For which story cases do need this level of nested pattern?

Reply:
line 148 `!src/js/**/**/*.stories.js`: covers stories under /context for example: `contexts/ResponsiveContext/responsive-context.stories.js`

line 149 `!src/js/**/**/stories/*.js`: covers stories in components other than Select

line 150 `!src/js/**/**/stories/**/**/*.js`: covers `components/Select/stories/components` and `components/Select/stories/components/icons`